src/test/java/org/folio/support/http/OkapiHeaders.java

```diff
@@ -1,10 +1,16 @@
 package org.folio.support.http;
 
+import lombok.AllArgsConstructor;
 import lombok.Value;
 
 @Value
+@AllArgsConstructor
 public class OkapiHeaders {
   String okapiUrl;
   String tenantId;
   String token;
+
+  public OkapiHeaders(OkapiUrl okapiUrl, String tenantId, String token) {
+    this(okapiUrl.toString(), tenantId, token);
+  }
```

Comment:
I don't understand this part.
Doesn't lombok's `@AllArgsConstructor` create a constructor like this already?
Why have both?

Reply:
Nevermind, I realized after I posted this that the type is `OkapiUrl` and the internal type is a `String`.